### DISCUSSION TITLE
Weird issue, I have to run forge clean after either edited my fuzz/invariant config

### DISCUSSION THREAD

**Discussion by Vincent71399** [Cyfrin/security-and-auditing-full-course-s23#228] [Q&A]
has anyone encountered the same issue, how did you guys resolve it? use forge clean && forge test xxx?

**Comment by EngrPips** [comment]
Yeah, I have seen people complain the same. I guess this is an issue with `Foundry`. `forge clean && forge test xxx` will be a good way to mitigate till Foundry officially fix it. Can you check if there is a reported case of this issue on the official `Foundry Repo`? If not, maybe you can make one.
